### ISSUE TITLE
How to configure proxy?

### ISSUE THREAD

**Issue by parag0kyndryl** [wictorwilen/express-msteams-host#23]
Hi, 

Can you tell me how can I configure the proxy while using @BotDeclaration?

Any help is appreciated.

Thank You.

**Comment by parag0kyndryl** [comment]
Basically, my server's network traffic must be routed via a proxy URL because I have a URL whitelisting in place. I'm just looking for a similar option while using this package.

Something like this: https://github.com/microsoftgraph/msgraph-sdk-javascript/blob/dev/src/IClientOptions.ts#L29 (I'm using this for Microsoft Graph API)

Edit: Updated the link

**Comment by wictorwilen** [comment]
@parag0kyndryl - the `BotDeclaration` only specifies the local endpoint  (path) of the server. It does not have anything to do on how things are routed to the server.

**Comment by parag0kyndryl** [comment]
You're right, but `express-msteams-host` internally trying to reach `https://login.botframework.com` and that is where the issue is. My server is inside a corporate network and without specifying a proxy option the package can not connect to `https://login.botframework.com`

**Comment by wictorwilen** [comment]
It's not `express-msteams-host` that is trying to do this, it's the bot framework and it is required for bot framework to communicate with that endpoint. This might help you: https://docs.microsoft.com/en-us/composer/configure-composer-proxy?tabs=windows
